java/query-map-image-sublayer/README.md

```diff
@@ -1,27 +1,37 @@
-# Query Map Image Sublayer
-Execute an attribute and spatial query on the sublayers of an ArcGIS map image layer.
+# Query map image sublayer
 
-Sublayers of an `ArcGISMapImageLayer` may expose a `ServiceFeatureTable` through a `Table` property. This allows you to perform the same queries available when working with a table from a `FeatureLayer`: attribute query, spatial query, statistics query, query for related features, and so on.
+Find features in a sublayer based on attributes and location.
 
-![Query Map Image Sublayer App](query-map-image-sublayer.png)
+![Image of query map image sublayer](query-map-image-sublayer.png)
+
+## Use case
+
+Sublayers of an `ArcGISMapImageLayer` may expose a `ServiceFeatureTable` through a `table` property. This allows you to perform the same queries available when working with a table from a `FeatureLayer`: attribute query, spatial query, statistics query, query for related features, etc. An image layer with a sublayer of counties can be queried by population to only show those above a minimum population.
 
 ## How to use the sample
-Specify a minimum population in the text box and click button to query the sublayers. After a short time, the results for each sublayer will appear as graphics.
+
+Specify a minimum population in the input field (values under 1810000 will produce a selection in all layers) and tap the query button to query the sublayers in the current view extent. After a short time, the results for each sublayer will appear as graphics.
 
 ## How it works
-To query map image sublayers:
-1. Create an `ArcGISMapImageLayer` using the URL of it's image service.
-1. After loading the layer, get its sublayers you want to query with `(ArcGISMapImageSublayer) layer.getSubLayers().get(index)`.
-1. Load the sublayer, and then get its `ServiceFeatureTable` with `sublayer.getTable()`.
-1. Create `QueryParameters`. You can use the where clause to query against a table attribute or set the parameters geometry to limit the results to an area of the map.
-1. Call `sublayerTable.queryFeaturesAsync(queryParameters)` to get a `FeatureQueryResult` with features matching the query.
-1. Iterate through the result features to use them however you wish. 
+
+1. Create an `ArcGISMapImageLayer` object using the URL of an image service.
+2. After loading the layer, get the sublayer you want to query with `(ArcGISMapImageSublayer) layer.getSubLayers().get(index)`.
+3. Load the sublayer, and then get its `ServiceFeatureTable` with `sublayer.getTable()`.
+4. Create `QueryParameters`. You can use `queryParameters.setWhereClause(sqlQueryString)` to query against a table attribute and/or set `queryParameters.setGeometry(extent)` to limit the results to an area of the map.
```

Comment:
```suggestion
4. Create a `QueryParameters` object. You can use `queryParameters.setWhereClause(sqlQueryString)` to query against a table attribute and/or set `queryParameters.setGeometry(extent)` to limit the results to an area of the map.
```

Reply:
It looks like we use complete sentences for the "How it works" section, but I could be wrong!

Reply:
yes LGTM!